app/views/lead-inspector/confirmation.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Confirmation page example
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <div class="govuk-panel govuk-panel--confirmation">
+        <h1 class="govuk-panel__title">Pre-inspection workflow completed</h1>
+        <div class="govuk-panel__body">
+          We've emailed your planning document to the provider
+        </div>
+      </div>
+
+      <p>
+        We have sent you a copy of that email.
+      </p>
+
+      <h2 class="govuk-heading-m">What happens next</h2>
+      <p>We will email you once the ITE partnership uploads their pre-inspection documents.</p>
+      <form action="dashboard" method="post" class="form">
+        {{ govukButton({
```

Comment:
I think a regular link instead of a button would be better here, as it's just going from one page to another and not submitting a form,  or beginning a transaction?